src/components/Sidebar/Sidebar.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
-import { Navbar } from './Navbar';
-import { Search } from './Search';
-import { Chats } from './Chats';
+import { Navbar } from '../Navbar/Navbar';
+import { Search } from '../Search/Search';
+import { Chats } from '../Chats/Chats';
 
```

Comment:
Elements exported by default, no need to parse them

Reply:
this is not mine=) i will change it